pkg/mcs/resource_manager/server/manager.go

```diff
@@ -121,6 +124,23 @@ func (m *Manager) Init(ctx context.Context) {
 		}
 	}
 	m.storage.LoadResourceGroupStates(tokenHandler)
+
+	// add default group
+	if _, ok := m.groups[defaultGroupName]; !ok {
```

Comment:
Is it possible that we overwrite some user-modified config here?

Reply:
Here load groups firstly, if user modified, the default group should already exists.